composer.json

```diff
@@ -16,7 +16,8 @@
         }
     ],
     "require": {
-        "php": ">=8.1"
+        "php": ">=8.1",
+        "phpunit/phpunit": "^9"
```

Comment:
phpunit is a development-time dependency. Move it to the require-dev section, please.